docs/docs/03-light-clients/04-wasm/09-migrations.md

```diff
@@ -9,6 +9,17 @@ slug: /ibc/light-clients/wasm/migrations
 
 This guide provides instructions for migrating 08-wasm versions.
 
+Please note that the following releases are retracted. Please refer to the appropriate migrations section for upgrading.
+
+```
```

Comment:
what language?

Reply:
Maybe just `bash` or something like that to keep it happy?